spec/views/rules_of_origin/_tab.html.erb_spec.rb

```diff
@@ -97,4 +105,30 @@
       expect(rendered_page).not_to have_css '.rules-of-origin-fta'
     end
   end
+
+  describe 'import trade summary duty box' do
```

Comment:
I don't think you need this describe and actually it doesn't follow better specs

https://www.betterspecs.org/#describe

Reply:
In general, I agree with using the method in the `describe`, anyway in this case we refer to a part of UI.
We have other examples in our codebase:

```
describe 'intro paragraph' do
...
describe 'goods nomenclature row' do
...
describe 'supplementary unit row' do
...
....
```

Reply:
Well, it also doesn't follow the example of the file its changing but fine :)